src/CopyrighFixer/CopyrighFixer/owner.cpp

```diff
@@ -20,12 +20,38 @@ const QString &Owner::getOwnerName() const {
     return _name;
 }
 
-void Owner::setTimeRange(const QString &interval) {
-    _timeRange = interval;
+void Owner::setTimePoint(const int &interval) {
+    _timePoint = interval;
 }
 
-const QString &Owner::getTimeRange() const {
-    return _timeRange;
+const int &Owner::getTimePoint() const {
+    return _timePoint;
+}
+
+void Owner::fromjson(const QJsonObject &objJs) {
+    if (objJs.contains("name") && objJs.value("name").isString()) {
+         _name = objJs.value("name").toString();
+    }
```

Comment:
You spent extra memory and time since you are performing the conversion 2 times (toString). 
if you use the value method the it makes no sense use the contains method because the value method check object key itself and return empty value if the object not exists. See Qt [documentation](https://doc.qt.io/qt-5/qjsonobject.html#value) about value method.
```suggestion
    _name = objJs.value("name").toString();
```